### PR TITLE
README.md updated

### PR DESCRIPTION
**Pull request by aslesarenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,123 @@
...
+
+## Repository Organization
+
+| sub-module  | description |
+|---|-----|
+| common | Used in all other submodules and contain basic utility classes |
+| core   | Implementation of graph-based intermediate representation of ErgoTree, which is used in cost estimation and interpretation |
+| docs   | Collection of documents |
+| library | Implementation of graph IR nodes for Coll, Size and other types  |
+| libaray-api | Declarations of interfaces   |
```
Comment (kushti) [inline README.md:104]: libaray => library

File: README.md [hunk 2]
```diff
@@ -1,45 +1,123 @@
...
+| libaray-impl | Implementation of interfaces   |
```
Comment (kushti) [inline README.md:105]: libaray => library

**Review by kushti — APPROVED** [review]
Approved, but please address comments

File: README.md [hunk 3]
```diff
@@ -1,45 +1,123 @@
...
- 3. In your own project add library dependency
- ```
-libraryDependencies ++= Seq(
-  "org.scorexfoundation" %% "sigma-state" % "0.9.4"
-) 
- ```
+# ErgoScript compiler and ErgoTree interpreter 
+
+This repository contains implementations of ErgoScript compiler and ErgoTree
+Interpreter for a family of Sigma-protocol authentication languages (or simply
```
[the remark this answers is not in the document]
Comment (aslesarenko) [reply README.md:4]: fixed

File: README.md (hunk 2 above)
Comment (aslesarenko) [reply README.md:105]: fixed

File: README.md (hunk 1 above)
Comment (aslesarenko) [reply README.md:104]: fixed

**Pull request merged**